### ISSUE TITLE
[Bug][SDK] Cannot setup webhook if `org_id` is not sent.

### ISSUE THREAD

**Issue by Dzeranov** [opencv/cvat#5857]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Steps to Reproduce (for bugs)

Follow this example: https://opencv.github.io/cvat/docs/api_sdk/sdk/reference/apis/webhooks-api/#create
but without setting `org_id` field.

### Expected Behaviour

No errors.

### Current Behaviour

`Invalid type for variable 'organization'. Required value type is int and passed type was NoneType at ['received_data']['organization']`
### Possible Solution

https://opencv.github.io/cvat/docs/api_sdk/sdk/reference/apis/webhooks-api/#create
https://opencv.github.io/cvat/docs/api_sdk/sdk/reference/models/webhook-read/
Despite the fact that `org_id` and `organization` fields are optional (and possible types are `int` and `none_type`) SDK is throwing error described above. Probably, the problem is with the API schema.

### Your Environment

- Git hash commit: 2c4676ace32a68572b38205f002a98e93af917d0
- Docker version `docker version`: 20.10.22
- Operating System and version: MacOS

**Comment by kirill-sizov** [comment]
Hi @Dzeranov, it seems that it's a problem in our documentation, you can create webhook without specifying org_id with the following script
```import time
from cvat_sdk.api_client import Configuration, ApiClient, exceptions
from cvat_sdk.api_client.model.webhook_read import WebhookRead
from cvat_sdk.api_client.model.webhook_write_request import WebhookWriteRequest
from pprint import pprint

configuration = Configuration(
    host = "<your_host>",
    username = '<username>',
    password = '<password>',
)

with ApiClient(configuration) as api_client:
    webhook_write_request = WebhookWriteRequest(
        target_url="http://examples.com",
        description="description_example",
        type="project",
        content_type="application/json",
        secret="secret_example",
        is_active=True,
        enable_ssl=True,
        project_id=1,
        events=[
            "create:comment",
        ],
    )

    try:
        (data, response) = api_client.webhooks_api.create(webhook_write_request)
        pprint(data)
    except exceptions.ApiException as e:
        print("Exception when calling WebhooksApi.webhooks_create: %s\n" % e)
```

**Comment by kirill-sizov** [comment]
Documentation problem has been resolved in #5903.
Valid example is [here](https://opencv.github.io/cvat/docs/api_sdk/sdk/reference/apis/webhooks-api/#create) (it works without org_id)
